src/custom/components/TransactionConfirmationModal/TransactionConfirmationModalMod.tsx

```diff
@@ -282,13 +282,15 @@ export function L2Content({
   hash,
   pendingText,
   inline,
+  isLimitOrderSubmit,
```

Comment:
I think we shouldn't mix limit-orders into generic `TransactionConfirmationModalMod.tsx` component, would be better to create a new component for this task.
However, for the current release it's ok :)

Reply:
agree 100%!